src/patternfly/components/Backdrop/styles.scss

```diff
@@ -0,0 +1,21 @@
+@import "../../patternfly-utilities";
+
+:root {
+  --pf-c-backdrop--ZIndex: var(--pf-global--ZIndex--xl);
+  --pf-c-backdrop--Color: rgba(0, 0, 0, .5);
```

Comment:
Can we add a comment about this value? I think we'll generate this as a global variable when we implement #266